CodeEdit/Features/Git/Client/GitClient+Clone.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  GitClient+Clone.swift
+//  CodeEdit
+//
+//  Created by Albert Vinizhanau on 10/20/23.
+//
+
+import Foundation
+import Combine
+
+extension GitClient {
+    /// Clone repository
+    /// - Parameters:
+    ///   - remoteUrl: URL of remote repository
+    ///   - localPath: Local path to clone
+    /// - Returns: Stream of progress
+    func cloneRepository(
+        remoteUrl: URL,
+        localPath: URL
+    ) -> AsyncThrowingMapSequence<LiveCommandStream, Double> {
+        let command = "clone \(remoteUrl.absoluteString) \(localPath.relativePath.escapedWhiteSpaces()) --progress"
+
+        return self.runLive(command)
+            .map { line in
+                // Parsing inspired by VS Code https://github.com/microsoft/vscode/blob/main/extensions/git/src/git.ts
```

Comment:
I think comments should be more informative by describing the code's functionality, rather than mentioning its inspiration source. What do you think?